scripts/api/api.py

```diff
@@ -0,0 +1,36 @@
+import requests
+import logging
+
+
+def create_data_file(api_url, data_file_data):
+    request_url = api_url + "day"
+    headers = {"content-type": "application/json"}
+    r = requests.post(
+        request_url,
+        headers=headers,
+        data=data_file_data,
+    )
+    status = r.status_code
+    if status != 200:
+        response = r.json()["message"]
+        logging.info(response)
+    else:
+        response = r.json()["data"]
+        logging.info(response)
+
+
+def create_metadata_file(api_url, metadata_file_data):
+    request_url = api_url + "metadata"
+    headers = {"content-type": "application/json"}
+    r = requests.post(
+        request_url,
+        headers=headers,
+        data=metadata_file_data,
+    )
+    status = r.status_code
+    if status != 200:
+        response = r.json()["message"]
+        logging.info(response)
+    else:
+        response = r.json()["data"]
+        logging.info(response)
```

Comment:
Unless I'm missing something, these look identical aside from the url's postfix. Could you move this functionality into a common function that both `create_data_file` and `create_metadata_file` both call?

Reply:
[655c34d](https://github.com/AMP-SCZ/dpimport/pull/17/commits/655c34d7cc922806cb0928f4d5234932a18bcfaf)